tests/miningPool/testMiningPoolMultiplePools.js

```diff
@@ -0,0 +1,52 @@
+let Web3 = require('aion-web3');
+let web3 = new Web3(new Web3.providers.HttpProvider("http://localhost:8545"));
+const spawn = require('child_process').spawn;
+const fs = require('fs');
+const assert = require('assert');
+const Cleanup = require('../cleanup');
+
+//we need to keep references so we will kill them at exit
+let miningPoolProcesses;
+let minerProcess1;
+let minerProcess2;
+let dockerComposeUpCommand = "docker-compose -f test-mining-pools.yml up -d";
+let dockerComposeDownCommand = "docker-compose -f test-mining-pools.yml down";
+
+let startMiningPools = () => {
+    miningPoolProcesses = spawn(testConfig.dockerComposeLocation + dockerComposeUpCommand, [], {detached: true});
+
+    miningPoolProcesses.on('error', function(err){
+        console.log(err);
+    });
+};
+
+let startMiners = () => {
+    //start first miner
+    let argument1 = ["-t", testConfig.miner1Threads, "-u", testConfig.miner1];
+    minerProcess1 = spawn(testConfig.aionminerCpuLocation, argument1, { detached: true});
+
+    minerProcess1.on('error', function(err){
```

Comment:
I find it difficult to find what's wrong just by logging errors. What do you think about redirecting `stderr` and `stdout` and seeing the actual logs for each subprocess(miner? Something like:
```
minerProcess1.stderr.on('data', function(data) {
    //you may need to check for multiple line endings here
    console.log(data);
});
```

Reply:
node & spawn will complain if we don't catch the error like this from what I remember

Reply:
You could leave the error handling and also add the output listeners afterwards.